src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -55,21 +59,17 @@ public class InternalUsersApiAction extends PatchableResourceApiAction {
             ":" // Not allowed in basic auth, see https://stackoverflow.com/a/33391003/533057
     );
 
+    public static final String LEGACY_OPENDISTRO_PREFIX = "_opendistro/_security";
+    public static final String PLUGINS_PREFIX = "_plugins/_security";
+
     private static final List<Route> routes = addRoutesPrefix(ImmutableList.of(
             new Route(Method.GET, "/user/{name}"),
             new Route(Method.GET, "/user/"),
+            new Route(Method.GET, "/user/serviceaccounts"),
+            new Route(Method.GET, "/user/internalaccounts"),
             new Route(Method.POST, "/user/{name}/authtoken"),
             new Route(Method.DELETE, "/user/{name}"),
-            new Route(Method.PUT, "/user/{name}"),
-
-            // corrected mapping, introduced in OpenSearch Security
-            new Route(Method.GET, "/internalusers/{name}"),
```

Comment:
why remove these new routes?

Reply:
okay. Since we are adding two new routes for `/users`, we should add them here for `/internalusers` too

Reply:
I corrected it. 

As it is a new endpoint, I left the ithe it onlyt only with the new mapping(/internalusers), as retrocompatibility wouldnt be an issue